homeassistant/components/alarm_control_panel/lupusec.py

```diff
@@ -0,0 +1,90 @@
+"""
+This component provides HA alarm_control_panel support for Lupusec System.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/alarm_control_panel.lupusec/
+"""
+
+from datetime import timedelta
+
+from homeassistant.components.alarm_control_panel import AlarmControlPanel
+from homeassistant.components.lupusec import LupusecDevice
+
+import homeassistant.util.dt as dt_util
+
+from homeassistant.const import (STATE_ALARM_ARMED_AWAY,
+                                 STATE_ALARM_ARMED_HOME,
+                                 STATE_ALARM_DISARMED)
+
+DEPENDENCIES = ['lupusec']
+DOMAIN = 'lupusec'
```

Comment:
This platform belongs to the alarm_control_panel domain. Please remove this constant here. Import a constant, not called `DOMAIN`, from the lupusec component, instead.